README.md

```diff
@@ -42,7 +42,7 @@ Use it in a playbook as follows:
 ```yaml
 - hosts: all
   roles:
-    - cloudalchemy.node-exporter
+    - ansible-node-exporter
```

Comment:
This was correct when using ansible-galaxy (which is recommended by ansible community). Role is published at https://galaxy.ansible.com/cloudalchemy/node-exporter